public/js/configs/development.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I feel like the configs still belong at the top-level. We could make a top-level `configs` directory. We'll probably end up wanting to put some other stuff in there, like the `port` that the server runs on.